packages/calypso-build/webpack/transpile.js

```diff
@@ -41,6 +43,7 @@ module.exports.loader = ( {
 				cacheIdentifier,
 				cacheCompression,
 				presets,
+				plugins: plugins || [],
```

Comment:
nit: could you just pass the raw `plugins` value, just like with `presets`? The `babel-loader` should accept an `undefined` value and treat it as empty-ish.

Reply:
Fixed in ef135252ebc